packages/web/config/feature-flag.ts

```diff
@@ -35,12 +35,7 @@ export const Announcement:
       isWarning: true,
     }
   : {
-      enTextOrLocalizationPath: "levana.description",
-      link: {
-        enTextOrLocalizationKey: "levana.link",
-        url: "https://trade.levana.finance/",
-        isExternal: true,
-      },
+      enTextOrLocalizationPath: "frontier.description",
```

Comment:
do we no longer need the link or its properties?

Reply:
It seems like no link on the banner this time.